src/fsharp/FSComp.txt

```diff
@@ -1498,6 +1498,7 @@ notAFunctionButMaybeDeclaration,"This value is not a function and cannot be appl
 3351,chkFeatureNotRuntimeSupported,"Feature '%s' is not supported by target runtime."
 3352,typrelInterfaceMemberNoMostSpecificImplementation,"Interface member '%s' does not have a most specific implementation."
 3353,chkFeatureNotSupportedInLibrary,"Feature '%s' requires the F# library for language version %s or greater."
+3360,parsEqualsMissingInTypeDefinition,"Unexpected token in type definition. Expected '='."
```

Comment:
I think "Unexpected token in type definition. Expected '=' after the type name" would be great, and even better (if it's possible to pass the type name) would be "after the type 'type-name'", where `type-name` is passed in from the parser.

But if it's not possible to pass in the name then that's fine. Otherwise this is looking good!

Reply:
I've added the type name as well, might as well!
I think I've found an error in the parser implementation of the spec to do with this ident though, which I'll write up below.